src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_TypeInference.cs

```diff
@@ -130,6 +130,93 @@ static void Main()
         }
 
         [Fact]
+        public void LocalVar_FlowAnalysis_01()
```

Comment:
>LocalVar_FlowAnalysis_01 [](start = 20, length = 24)

It would be interesting to see what the behavior is for this scenario
```
    static void F(string? s)
    {
        t = null;
        var t = s;
        t.ToString();
    }

```

Reply:
Added test.